packages/onboarding/src/sensei-step-container/styles.scss

```diff
@@ -0,0 +1,27 @@
+.sensei {
```

Comment:
Note that this style bundle might persist through their session in Calypso. So if you have anything else named "sensei" now or in future, you might have CSS collision. Something more unique like `sensei-onboarding` might make sense.

I've recently noticed this kind of problems with signup form where we used a lot of classes like `.woo`, `.videopress` or `.jetpack`.

Not a blocker!

Reply:
I agree however this class comes from the stepper API and I think that we will need to rename the whole flow if we don't want to add an extra class. Do you think that it's ok to leave it as it is?